src/main/java/goodgenerator/blocks/tileEntity/base/LargeFusionComputer.java

```diff
@@ -146,6 +148,15 @@ public boolean allowCoverOnSide(ForgeDirection side, GT_ItemStack aStack) {
         return side != getBaseMetaTileEntity().getFrontFacing();
     }
 
+    @Override
+    public void loadNBTData(NBTTagCompound aNBT) {
+        super.loadNBTData(aNBT);
+        // Migration code
+        if (lEUt > 0) {
+            lEUt = -lEUt;
```

Comment:
Shouldn't you also migrate mEUt as it was using that before hand?

Reply:
No that is handled by TT. It loads lEUt with the key "mEUt"